doc/classes/XRInterface.xml

```diff
@@ -69,13 +69,38 @@
 				Is [code]true[/code] if this interface has been initialised.
 			</description>
 		</method>
+		<method name="is_passthrough_enabled">
+			<return type="bool" />
+			<description>
+				Is [code]true[/code] if passthrough is enabled.
+			</description>
+		</method>
+		<method name="is_passthrough_supported">
+			<return type="bool" />
+			<description>
+				Is [code]true[/code] if this interface supports passthrough.
+			</description>
+		</method>
 		<method name="set_play_area_mode">
 			<return type="bool" />
 			<param index="0" name="mode" type="int" enum="XRInterface.PlayAreaMode" />
 			<description>
 				Sets the active play area mode, will return [code]false[/code] if the mode can't be used with this interface.
 			</description>
 		</method>
+		<method name="start_passthrough">
+			<return type="bool" />
+			<description>
+				Starts passthrough, will return [code]false[/code] if passthrough couldn't be started.
+				[b]Note:[/b] The main viewport must have transparent background, otherwise passthrough may not properly render.
```

Comment:
Change this to `The viewport used for XR must have a transparent background...`

While on Quest this will always be your main viewport, on desktop it's just as likely that we use a second viewport for XR rendering and leave the main viewport for rendering a different view to screen. 
While today we do not have any desktop VR headsets that support passthrough, this may change in the future.